### PR TITLE
HSD8-751 Allow cron to invalidate cache tags

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Cron right now doesn't actually clear any varnish tags unless its ran through the UI. This fixes that
- Increments the release.

# Need Review By (Date)
- asap

# Urgency
- medium

# Steps to Test
1. if its not clear, i can show on my local.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: acquia_purge_d8cache.info [hunk 1]
```diff
@@ -2,5 +2,5 @@ name = Acquia Purge D8 Cache
 description = Purge Acquia varnish caches using D8Cache module, which is a backport of D8 cache tag mechanism
 core = 7.x
 package = Stanford - ACSF
-version = 7.x-1.3
+version = 7.x-1.4
```
Comment (sherakama) [inline acquia_purge_d8cache.info:5]: CHANGELOG and README need updates.

File: acquia_purge_d8cache.module [hunk 2]
```diff
@@ -162,7 +162,7 @@ function acquia_purge_d8cache_cron() {
       }
     }
   }
-
+  drupal_static(__FUNCTION__, TRUE);
```
Comment (sherakama) [inline acquia_purge_d8cache.module:165]: Seems reasonable.

**Comment by jbickar** [comment]
What's the use case here, like an event moving from "upcoming events" to "past events" and the cache tags for the Views getting invalidated on cron?

**Comment by pookmish** [comment]
@jbickar thats exactly the case. We had the cron job logic in place for that, but when we restricted the logic to not clear varnish during database updates, we prevented the cron job from working through CLI. It currently works when running cron through the UI.

File: acquia_purge_d8cache.info (hunk 1 above)
Comment (pookmish) [reply acquia_purge_d8cache.info:5]: :heavy_check_mark:

**Review by sherakama — APPROVED** [review]

**Comment by sherakama** [comment]
I didn't merge to allow @jbickar to comment before it gets green lit.

**Comment by jbickar** [comment]
OK, so it invalidates the cache tags on cron, but I'm not seeing where `_acquia_purge_d8cache_curl_request()` is called during cron? (Those are two separate things, right? Invalidating the tags, vs. making the curl requests to the load balancers?)

**Comment by pookmish** [comment]
1. `acquia_purge_d8cache_cron()` is executed which calls the function `drupal_invalidate_cache_tags()`
1. `drupal_invalidate_cache_tags()` calls the hook `hook_invalidate_cache_tags()` which is implemented by `acquia_purge_d8cache_invalidate_cache_tags()`
1. `acquia_purge_d8cache_invalidate_cache_tags()` function calls `_acquia_purge_d8cache_curl_request()`

**Comment by jbickar** [comment]
Ahhh. I knew I was missing something.

**Review by jbickar — APPROVED** [review]

**Pull request merged**